src/olympia/devhub/forms.py

```diff
@@ -456,14 +456,27 @@ def _clean_upload(self):
                          u'Please try again.'))
 
 
-class StandaloneValidationForm(AddonUploadForm):
-    is_unlisted = forms.BooleanField(
-        initial=False,
-        required=False,
-        label=_(u'Do not list my add-on on this site'),
-        help_text=_(
-            u'Check this option if you intend to distribute your add-on on '
-            u'your own and only need it to be signed by Mozilla.'))
+class StandaloneValidationForm(AddonUploadForm):      
+    LISTED_LABEL = _(
+        u'On this site. <span class="helptext">'
+        u'Your submission will be listed on this site and the Firefox '
+        u'Add-ons Manager for millions of users, after it passes code '
+        u'review. Automatic updates are handled by this site. This '
+        u'add-on will also be considered for Mozilla promotions and '
+        u'contests. Self-distribution of the reviewed files is also '
+        u'possible.</span>')
+    UNLISTED_LABEL = _(
+        u'On your own. <span class="helptext">'
+        u'Your submission will be immediately signed for '
+        u'self-distribution. Updates should be handled by you via an '
+        u'updateURL or external application updates.</span>')
+
+    channel = forms.ChoiceField(
+        choices=(
+            ('listed', mark_safe_lazy(LISTED_LABEL)),
+            ('unlisted', mark_safe_lazy(UNLISTED_LABEL))),
+            initial='listed' ,
+        widget=forms.RadioSelect(attrs={'class': 'channel'}))
```

Comment:
@eviljeff should I use `DistributionChoiceForm` instead of `StandaloneForm` in the views if it doesn't makes much of difference? Its used only in addon validator and compatibility checker.

Reply:
You can try to use DistributionChoiceForm and see if it works.  I thought you'd need the upload parts of AddonUploadForm but I forget how standalone validation works so you may not.